app/classifier/index.cjsx

```diff
@@ -156,80 +192,85 @@ Classifier = React.createClass
       onChange: -> classification.update()
 
     <div className="task-container" style={disabledStyle if @state.subjectLoading}>
-      {persistentHooksBeforeTask.map (HookComponent) =>
-        <HookComponent {...taskHookProps} />}
-
-      <TaskComponent autoFocus={true} taskTypes={tasks} workflow={@props.workflow} task={task} preferences={@props.preferences} annotation={annotation} onChange={@handleAnnotationChange.bind this, classification} />
+      {if @state.renderIntervention
```

Comment:
If a project is setup for an intervention, is there a chance volunteers won't see a summary for a subject? If so, they miss a chance to go to the subject's talk page too. Why take the classification summary space for the intervention instead of a dialog for instance?